src/cpl/nuopc/lnd_import_export.F90

```diff
@@ -1076,22 +1100,25 @@ subroutine state_getimport_2d(state, fldname, ctsmdata, rc)
   end subroutine state_getimport_2d
 
   !===============================================================================
-  subroutine state_setexport_1d(state, fldname, ctsmdata, minus, rc)
+  subroutine state_setexport_1d(state, fldname, ctsmdata, minus, init_spval, rc)
 
     ! fill in ctsm export data for 1d field
 
-    use ESMF, only : ESMF_LOGERR_PASSTHRU, ESMF_END_ABORT, ESMF_LogFoundError
-    use ESMF, only : ESMF_Finalize
+    use ESMF          , only : ESMF_LOGERR_PASSTHRU, ESMF_END_ABORT, ESMF_LogFoundError
+    use ESMF          , only : ESMF_Finalize
+    use shr_const_mod , only : shr_const_spval
 
     ! input/output variabes
     type(ESMF_State) , intent(in) :: state
     character(len=*) , intent(in) :: fldname
     real(r8)         , intent(in) :: ctsmdata(:)
     logical, optional, intent(in) :: minus
+    logical, optional, intent(in) :: init_spval
```

Comment:
- [ ] Could you please add a comment describing when you should and shouldn't set init_spval to true, to provide guidance for anyone adding fields in the future?

- [ ] [OPTIONAL] I would prefer init_spval to be a required argument, so that anyone adding a new field needs to explicitly think about whether it should be true or false. Otherwise I think it is likely to be left as implicitly false in situations where it really should be true.

Reply:
Sounds good; thanks. So I guess the long-term plan would be to not bother having this argument at all, but instead implicitly assuming it is true for all fields. In that case, I don't have strong feelings about whether this is left as an optional argument or turned into a required argument for now (it all depends on how long we expect it to be before we're able to achieve the long-term plan).

Reply:
@billsacks - I've incorporated the changes you suggested and am currently testing them. Please hold of on more testing until I've done that.

Reply:
Thanks, this looks good now, with the one remaining thing adding a brief comment about init_spval (though I'm okay skipping that if you don't feel you have a good enough understanding of the issue yet to add a meaningful comment). Once you give me the go ahead, I can do final testing.